File: letter/ru/Ш.py
from common import *

expr = Eq(
    x * (Abs(x) - S(1) / 2) * (y + 1)
    * sqrt(
        (S(1) / 2 + eps - Abs(x)) * (1 - y) /
        (y + 1 + eps)
    )
    ,
    0
)
size = 1

File: letter/ru/Ш.js
import { Letter } from '../../Letter.js';

const expr = 'x * (abs(x) - 1/2) * (y + 1) * sqrt((1/2 + 1/10 - abs(x)) * (1 - y) / (y + 1 + 1/10))';

export const letter = new Letter(expr, 1);
